src/test/app/MultiSign_test.cpp

```diff
@@ -1134,6 +1134,80 @@ class MultiSign_test : public beast::unit_test::suite
         env.close();
     }
 
+    void test_signForHash()
+    {
+        // Make sure that the "hash" field returned by the "sign_for" RPC
```

Comment:
Only the last `sign_for` before submitting returns the "right" hash huh? In general, each signer doesn't know the hash to look out for though right?

Reply:
Correct.  Each added signature changes the hash.  But if the transaction with those signatures is submitted, either as a blob through `submit` or as text through `submit_multisigned`, then the hash does not change.

As a use model, it is probably smartest to rely on the hash returned when the transaction is submitted.  But since we are returning a hash value on signing, the value we return should be correct.